cmd/abapEnvironmentPullGitRepo.go

```diff
@@ -154,3 +153,50 @@ func triggerPull(repositoryName string, pullConnectionDetails abaputils.Connecti
 	uriConnectionDetails.URL = body.Metadata.URI + expandLog
 	return uriConnectionDetails, nil
 }
+
+func checkPullRepositoryConfiguration(options abapEnvironmentPullGitRepoOptions) error {
+	if len(options.RepositoryNames) > 0 && options.Repositories != "" {
+		log.Entry().Info("It seems like you have specified both the repositories to be pulled as an in-line configuration as well as in the dedicated repositories configuration file.")
+		log.Entry().Info("Please note that in this case both configurations file will be handled and pulled.")
```

Comment:
[Minor] Is there a reason for two log Entries?
[Major] "in-line" is misleading
```suggestion
		log.Entry().Info("It seems like you have specified repositories directly via the configuration parameter "repositoryNames" as well as in the dedicated repositories configuration file. Please note that in this case both configurations will be handled and pulled.")
```

Reply:
Changed it